packages/components/src/utils/test/math.js

```diff
@@ -37,6 +37,27 @@ describe( 'subtract', () => {
 	} );
 } );
 
+describe( 'clamp', () => {
+	it( 'should clamp a value between min and max', () => {
```

Comment:
Since all tree arguments are optional/have default fallback values, should we add tests for when some/all arguments are not provided?